extensions/import/src/wizard/pages/prosePreviewPage.ts

```diff
@@ -84,7 +91,7 @@ export class ProsePreviewPage extends ImportPage {
 			await this.populateTable([], []);
 			this.isSuccess = false;
 			if (this.form) {
-				this.resultTextComponent.value = this.failureTitle;
+				this.resultTextComponent.value = this.failureTitle + '\n' + (error ?? '');
```

Comment:
I would use `os.EOL` here instead

Reply:
Curious if that matters when we're displaying text inside ADS? I thought \r\n might actually break the text rendering.

Reply:
You might be right, it might break layering.